ipwb/replay.py

```diff
@@ -717,7 +717,7 @@ def handler(signum, frame):
     for idx, hLine in enumerate(hLines):
         k, v = hLine.split(': ', 1)
 
-        if k.lower() == 'transfer-encoding' and v.lower() == 'chunked':
+        if k.lower() == 'transfer-encoding' and 'chunked' in v.lower():
```

Comment:
Yes it will, but I intentionally did not harden it to make it unnecessarily more complex than it should be, because there are only a few well-known transfer-encoding values that include `chunked`, `compress`, `deflate`, `gzip`, and `identity` (where the last one is the default and it means no change). When two or more values are combined, `chunked` is supposed to be the last value in the sequence (if present). While there is no one stopping someone to define more transfer encodings, but it is very unlikely to have something that has `chunked` as a substring and gets wider agent support.

That said, if you want, I can make it more specific to only match exactly `chunked`.

Reply:
5f90ce4 should take care of it.